code/API_definitions/qod-provisioning.yaml

```
@@ -69,44 +69,43 @@ externalDocs:
   url: https://github.com/camaraproject/QualityOnDemand
 
 servers:
-  - url: "{apiRoot}/qod-provisioning/vwip"
+  - url: "{apiRoot}/qos-provisioning/vwip"
     variables:
       apiRoot:
         default: http://localhost:9091
         description: API root, defined by the service provider, e.g. `api.example.com` or `api.example.com/somepath`
 tags:
-  - name: QoD Provisioning
-    description: Manage the permanent provisioning of QoD
+  - name: QoS Provisioning
+    description: Manage the permanent assignment of a QoS profile to a device
 
 paths:
-  /device-qos:
+  /qos-assignments:
     post:
       tags:
-        - QoD Provisioning
-      summary: Triggers a new provisioning of QoS for a device
+        - QoS Provisioning
+      summary: Triggers the provisioning process for the assignment of a QoS profile to a device
       description: |
-        Triggers the provisioning in the operator's environment to associate a given QoS profile with a given device.
+        Initiates the assignment of a specific QoS profile to a given device in the operator's environment.

```

I don't quite understand what is meant by "in the operator's environment"
```suggestion
        Initiates the assignment of a specific QoS profile to a given device when connected to the operator's network.
```

---

Fine with using network